lib/advanced-open-file-view.coffee

```diff
@@ -167,6 +176,13 @@ class AdvancedFileView extends View
     @miniEditor.setText newPath
 
   update: ->
+    if atom.config.get 'advanced-open-file.helmDirSwitch'
+      text = @miniEditor.getText()
+      if text.endsWith(path.sep + path.sep)
+        @updatePath @FS_ROOT
+      else if text.endsWith("~#{path.sep}")
+        @updatePath osenv.home() + path.sep
+
```

Comment:
There isn't, but I think making undo work inside the textbox is a good idea and I'll submit a followup PR with that feature.